### PR TITLE
update swagger

### PR DESCRIPTION
**Pull request by vicancy**
1. add `operationId`
2. remove duplicate actions
3. add more response details
4. add `head` action

### CODE REVIEW

File: docs/swagger/v1.json [hunk 1]
```diff
@@ -1,710 +1,690 @@
...
-            "content": {
-              "application/json": {
-                "schema": {
-                  "$ref": "#/components/schemas/PayloadMessage"
-                }
-              }
+          }
+        },
+        "responses": {
+          "200": {
```
Comment (vwxyzh) [inline docs/swagger/v1.json]: should be `202`

File: docs/swagger/v1.json [hunk 2]
```diff
@@ -1,710 +1,690 @@
...
+          "503": {
+            "description": "The service is temporarily unavailable"
+          }
+        }
+      }
+    },
+    "/api/v1/hubs/{hub}": {
+      "post": {
+        "tags": [
+          "HubProxyControllerV1"
```
Comment (JialinXin) [inline docs/swagger/v1.json]: shall we hide this?

File: docs/swagger/v1.json [hunk 3]
```diff
@@ -1,710 +1,690 @@
...
-              }
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "Success"
+          }
+        }
+      },
+      "get": {
```
Comment (vwxyzh) [inline docs/swagger/v1.json]: not released...

File: docs/swagger/v1.json [hunk 4]
```diff
@@ -1,710 +1,690 @@
...
-              "description": "Close reason.",
-              "schema": {
-                "type": "string"
-              }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success"
+          },
+          "400": {
```
Comment (vwxyzh) [inline docs/swagger/v1.json]: should we give 400 in swagger file?

File: docs/swagger/v1.json [hunk 5]
```diff
@@ -1,710 +1,690 @@
...
+            "description": "Bad Request"
+          },
+          "404": {
+            "description": "Not Found"
+          },
+          "500": {
```
Comment (vwxyzh) [inline docs/swagger/v1.json]: should we give `500` in swagger file?

File: docs/swagger/v1.json (hunk 2 above)
Comment (vicancy) [reply docs/swagger/v1.json]: any need to ?

Comment (vicancy) [reply docs/swagger/v1.json]: > Tagged operations may be handled differently by tools and libraries. For example, Swagger UI uses tags to group the displayed operations

Sounds useful

File: docs/swagger/v1.json (hunk 3 above)
Comment (vicancy) [reply docs/swagger/v1.json]: will on-hold the change until the new rt is deployed

File: docs/swagger/v1.json (hunk 4 above)
Comment (vicancy) [reply docs/swagger/v1.json]: makes sense

File: docs/swagger/v1.json (hunk 2 above)
Comment (JialinXin) [reply docs/swagger/v1.json]: shall we tag with some meaningful names instead of internal controller name, which sounds not useful to customer.

Comment (vicancy) [reply docs/swagger/v1.json]: updated

File: docs/swagger/v1.json [hunk 6]
```diff
@@ -1,710 +1,975 @@
...
+        "tags": [
+          "general"
+        ],
+        "summary": "Get service health status.",
+        "operationId": "HealthApi_Head_Index",
+        "responses": {
+          "200": {
+            "description": "The service is healthy"
+          },
+          "503": {
```
Comment (wanlwanl) [inline docs/swagger/v1.json:19]: also 502? https://github.com/Azure/azure-signalr/blob/dev/src/Microsoft.Azure.SignalR.Management/ServiceManager.cs#L168

Comment (vicancy) [reply docs/swagger/v1.json:19]: swagger describes the service behavior, it does not include cases when service/network is down

Comment (vicancy) [reply docs/swagger/v1.json:19]: yeah you are right we missed several expected status code

Comment (vicancy) [reply docs/swagger/v1.json:19]: I feel we don't need to list 5xx code, every API can encounter 5xx. And for the client side, it should consider all 5xx code as `isHealthy = false`

Reference: https://stackoverflow.com/a/58277444/687394

**Pull request merged**